metrics/core/lib/src/data/model/build_data.dart

```diff
@@ -25,12 +27,44 @@ class BuildData extends Build implements DataModel {
           coverage: coverage,
         );
 
+  /// The unique identifier of this build.
+  String get documentId => '${projectId}_$id';
```

Comment:
```suggestion
  String get documentId => '${projectId}_$buildNumber';
```
It seems like we decided to use the build number as the second part of this value, didn't we? 

cc @illia-romanenko

Reply:
Can we summarize our discussion about adding separate getter for documentId? vs setting id with projectId + buildNumber?

Reply:
Yes, we can get rid of this getter and use the `id` field. This makes sense since we are using [BuildData.id] as a document ID in Metrics. Thanks, I missed that in review!